website/registrations/tests/test_views.py

```diff
@@ -86,7 +86,8 @@ def _give_user_permissions(self):
     def test_permissions(self):
         url = f"/registration/admin/process/{self.entry1.pk}/"
         response = self.client.post(url)
-        self.assertRedirects(response, f"/admin/login/?next={url}")
+        self.assertEqual(response.status_code, 302)
```

Comment:
Why do this? assertRedirects assumes 302 by default...

Reply:
I changed the admin login view to redirect to the regular login view, which redirects again. So it's a 302 to a 302. Idk but without it, the tests failed